src/main/js/encounterDrawer.js

```diff
@@ -87,9 +88,36 @@ class EncounterDrawerUnstyled extends React.Component {
     this.refreshCombatantsState();
   }
 
-  updateCombatant(combatant) {
-    //TODO patch call
-    this.refreshCombatantsState();
+  updateCombatant(url, data) {
+      fetch(url, {
+          method: 'PATCH',
+          body: JSON.stringify(data),
+          headers: {
+            'Content-Type': 'application/json'
+          }
+      }).catch(err => err)
+      .then(() => this.refreshCombatantsState());
+  }
+
+  handleChange(combatantId, dataType, value) {
```

Comment:
can you move the handle change to the combatantList component and then pass the updateCombatant in?

Reply:
Trying to isolate the combatantUpdate operations and the individual component flow work.  Considering adding redux to simplify state management longer term

Reply:
If move the handleChange to the combatantList file, then how do I setState
    this.setState({
       combatants: newState
    });

is there a built in function that i can update parent state from child?